newsrevenuehub/rev-engine
# [NRH-152]  Org Accept or Reject Contribution

#### What's this PR do?
From the Donation detail view, accessed by clicking on a donation in the donation list view, you can now accept or reject a flagged contribution.

#### How should this be manually tested?
If necessary, ensure that a could valid flagged, not auto-accepted contributions exist. Then visit /dashboard/donations and click on that donation in the list.  Test accepting and rejecting them.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
